nvtabular/ops.py

```diff
@@ -1102,3 +1102,43 @@ def op_logic(self, gdf: cudf.DataFrame, target_columns: list, stats_context=None
             new_gdf[col] = self.f(gdf[col], gdf)
         new_gdf.columns = [f"{col}_{self._id}" for col in new_gdf.columns]
         return new_gdf
+
+
+class Filter(TransformOperator):
+    """
+    Enables to call Methods to cudf.Series. Returns a subset of the original gdf.
+
+    Parameters
+    -----------
+    f : lambda function
+        defines the function executed on dataframe level, expectation is lambda gdf: gdf[gdf.col...]
+        col is the cudf.Series defined by the context
+        gdf is the full cudf.DataFrame
+    preprocessing : bool, default True
+        Sets if this is a pre-processing operation or not
+    replace : bool, default True
+        Replaces the transformed column with the original input
+        if set Yes
```

Comment:
I realize that ```replace``` is needed by the base class, but I don't think it makes sense in this context (like replace=False doesn't create new columns). Maybe just remove as a method parameter and always pass replace=True to the base class?